src/components/overview/ImageGallery/DefaultView.jsx

```diff
@@ -1,43 +1,26 @@
 import React, { useState, useEffect } from 'react';
 
-const DefaultView = ({state, updateState}) => {
-  const [ mainImage, setMainImage ] = useState('');
-
-  const updateMainImage = (event) => {
-    event.preventDefault();
-    let clickedImg = event.target.src;
-    setMainImage(clickedImg);
-  };
-
-  const handleToExpand = () => {
-    let mainImageElement = document.getElementById('main');
-    mainImageElement.style.width = '75rem';
-  };
+const DefaultView = ({state, updateState, updateMainImage, handleToExpand, handleRightArrow, handleLeftArrow}) => {
 
   return (
-    <div className='defaultView'>
-      <div className='styleImages'>
-        {state.styleImages.map((element, idx) => { //render all style photos
-          return <img
-            style = {{
-              width: '9vh',
-              height: '9vh',
-              cursor: 'pointer'
-            }}
-            id = {idx}
-            onClick = {updateMainImage}
-            src={element.url}/>;
-        })}
-      </div>
-      <div className='mainImage'>
-        <img className='btnFloating' id='left' src ="https://img.icons8.com/ios-filled/50/000000/long-arrow-left.png"></img>
-        {(state.styleImages[0] !== undefined && mainImage === '') ?
-          <img id = 'main' onClick = {handleToExpand} src = {state.styleImages[0].url} style = {{width: '75vh', height: '54vh'}}/> :
-          <img id = 'main' onClick = {handleToExpand} src = {mainImage} style = {{width: '75vh', height: '54vh'}}/>
-        }
-        <img className='btnFloating' id='right' src ="https://img.icons8.com/ios-filled/50/000000/long-arrow-right.png"></img>
-      </div>
-    </div>
+    <>
+      {(state !== undefined) && (
+        <div className='defaultView'>
+          <div className='styleImages'>
+            {state.styleImages.map((element, idx) => { //render all style photos
+              return <img id={idx} key={idx} className='eachStyleImage' onClick={updateMainImage} src={element.url}/>; })}
+          </div>
+          <div className='mainImage' id='mainImage'>
+            <img onClick={handleLeftArrow} className='btnFloating' id='left' src ="https://img.icons8.com/ios-filled/50/000000/long-arrow-left.png"></img>
+            {(state.styleImages[0] !== undefined && state.mainImage === '') ?
+              <div onClick = {handleToExpand} id='wrap'><img id = 'main' src = {state.styleImages[0].url}/></div> :
+              <div onClick = {handleToExpand} id='wrap'><img id = 'main' src = {state.mainImage}/></div>
```

Comment:
Tiny visual thing here- line 16 & 17- is there a reason for the added spaces for the `id = 'main' src =` ?

Reply:
Sorry no specific reason for this. I deleted the spaces!